configuration/environment_variables.rst

```diff
@@ -445,6 +445,47 @@ Symfony provides the following env var processors:
                 'auth' => '%env(file:AUTH_FILE)%',
             ]);
 
+``env(require:FOO)``
+    ``require()`` the PHP file whose path is the value of the ``FOO``
+    env var and return the value ``return``ed from it.
```

Comment:
This is not valid reStructuredText. Let's just do `returned` as a word

Reply:
fixed